oracle/types/errors.go

```diff
@@ -12,9 +12,11 @@ var (
 	ErrMissingExchangeRate = sdkerrors.Register(ModuleName, 2, "missing exchange rate for %s")
```

Comment:
I was worried about updating the existing error codes but that looks fine. However, when reading the docs Custom error codes `Must be greater than one, as a code value of one is reserved for internal errors.` (https://docs.cosmos.network/main/building-modules/errors) Maybe the error codes should start at 2?

Reply:
Good catch! Updated them to start at 2 👍.